src/app/Http/Controllers/Operations/ListOperation.php

```diff
@@ -27,11 +27,13 @@ protected function setupListRoutes($segment, $routeName, $controller)
             'operation' => 'list',
         ]);
 
-        Route::get($segment.'/{id}/details', [
-            'as'        => $routeName.'.showDetailsRow',
-            'uses'      => $controller.'@showDetailsRow',
-            'operation' => 'list',
-        ]);
+        if (! isset($this->setupDetailsRowRoutes) || $this->setupDetailsRowRoutes === true) {
```

Comment:
Why add a new way to configure things, as a property on CrudController? 

All other configurations are done using `CRUD::set()`, what makes this one so different, that it requires a different paradigm?

Reply:
So it sounds like a good way to do it, yes.

To properly support this property, let's:
- [ ] add this property to the ListOperation trait, `public setupDetailsRowRoute = true`;
- [ ] no longer test if it exits, only if it's true or false;
- [ ] let's make the property singural instead of plural (`setupDetailsRowRoute`, not `setupDetailsRowRoutes`); that way it's different than the setup route methods;

That way I think it's more clear, the trait comes with the property.

Reply:
Good job here Pedro, I really couldn't think of a better way out of this hole.

Reply:
Ok great.